src/pages/TextMessages.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div v-if="!loading">
+    <v-flex xs12 sm6 offset-sm3>
+      <v-card class="messages-container">
+
+        <v-card-title class="headline">
+          Text Messages
+        </v-card-title>
+
+        <v-card-text>
+
+          <TextMessageViewer
+            class="message-viewer"
+            v-for="(message, key) in messages"
+            :message="message"
+            :key="key"
+          />
+
+        </v-card-text>
+      </v-card>
+    </v-flex>
+
+    <v-flex xs12 sm6 offset-sm3
+            :class="composerIsSticky ? 'sticky-composer' : ''">
+      <TextMessageComposer
+        :icon="composerIsSticky ? 'arrow_drop_down' : 'arrow_drop_up'"
+        @send="send"
+        @iconClicked="composerIsSticky = !composerIsSticky"
+      />
+    </v-flex>
+  </div>
+</template>
+
+<script>
+import {textMessages} from '../firebase'
+import {currentUser} from '../main'
```

Comment:
Läsa inte currentUser kanske jag också borde göra då, läste nog in den själv i min komplexa data.

Reply:
Gör så att individuella komponenter inte behöver lägga till ytterliggare onAuthChanged listeners. Behöver bara underhålla en onAuthChanged listener.